renderer/components/Util/WalletName.js

```diff
@@ -8,7 +8,8 @@ const WalletName = ({ wallet }) => {
   if (wallet.type === 'local') {
     return wallet.name || `Wallet #${wallet.id}`
   }
-  return wallet.name || wallet.host.split(':')[0]
+  const { name, host } = wallet
```

Comment:
Could you please move destructuring up, since `wallet.name` is used 2 lines above ?

Reply:
Updated